src/main/java/calculator/StringCalculator.java

```diff
@@ -0,0 +1,63 @@
+package calculator;
+
+import java.math.BigInteger;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Pattern;
+
+public class StringCalculator {
+
+    private static final String CUSTOM_DELIMITER_START = "//";
+    private static final String CUSTOM_DELIMITER_END = "\\n";
+
+    ArrayList<String> delimiters = new ArrayList<>(List.of(",", ":"));
+
+    public BigInteger calculate(String inputString) {
+        inputString = extractDelimiter(inputString);
+
+        String regex = String.join("|", delimiters);
+        String[] splitByDelimiter = inputString.split(regex);
+
+        return sumNumbers(splitByDelimiter);
+
+    }
+
+    private String extractDelimiter(String inputString) {
+        if (inputString.startsWith(CUSTOM_DELIMITER_START)) {
+            if (inputString.contains(CUSTOM_DELIMITER_END)) {
```

Comment:
```suggestion
if (inputString.startsWith(CUSTOM_DELIMITER_START) && inputString.contains(CUSTOM_DELIMITER_END)) {
```
[제안] if문을 중첩해서 쓰는 것보다 한번에 묶고 예외는 따로 if문으로 처리하는 것이 좋을 것 같아요!